index.html

```diff
@@ -1492,6 +1492,62 @@ <h3>Types</h3>
 
       </section>
 
+      <section>
+        <h3>Names and Descriptions</h3>
+
+        <p>
+When displaying a <a>credential</a>, it can be useful to have
+text provided by the <a>issuer</a> that provides the
+<a>credential</a> with a name as well as a short description of the
+purpose of the <a>credential</a>. The `name` and `description` <a>properties</a>
+are provided for these purposes.
+        </p>
+
+        <dl>
+          <dt><dfn class="lint-ignore">name</dfn></dt>
```

Comment:
why lint ignore?

Reply:
The term is not /internally/ referenced anywhere else in the specification, and if ReSpec detects that, it throws warnings, which triggers Echidna to not publish the specification (because there are warnings). 

We do reference this term externally, from the vocabulary, so we want to keep the DFN and we use `lint-ignore` to silence the ReSpec warning that's thrown (by telling ReSpec that we really do mean to define the term).

Reply:
would it be better to just add a single internal reference? (off topic, and not blocking)

Reply:
I guess we could add an internal reference (in a future PR). I usually make a pass right before Proposed Rec to clean up the "lint-ignore" concerns. There is also the possibility of exporting the term (so it's added to the xref glossary, though I imagine adding `name` and `description` to the xref glossary would be frowned upon since they're so generic).